### DISCUSSION TITLE
Add bookmark feature for directories or files.

### DISCUSSION THREAD

**Discussion by RMNB** [filebrowser/filebrowser#1545] [Ideas]
I often use filebrowser to edit certain config files. I'm always frustrated when I drill down through directories to get to them. Would it be possible to add a bookmarks/favorites feature to the sidebar? I've considered making bookmarks or symbolic links for the directories I often use but a built in bookmark feature would be great.

**Comment by pewter77** [comment]
Would also like this. Favorite Folders feature would be great to have.

**Reply by Sledgehamma** [reply]
Yes, that would indeed be a great feature.

**Reply by hernandito** [reply]
+1 here as well...

**Comment by hernandito** [comment]
Hi... After figuring out how to recompile the code with my edits,  I was able to hard code bookmarks into my version of filebrowser (it could use some visual improvements): 


![image](https://user-images.githubusercontent.com/787317/147605902-53bc8653-6c0f-407f-bc2b-fdcfeeaf3498.png)


I basically edited the file **_Sidebar.vue_** to include something like this:

          `        <span>{{ $t("sidebar.signup") }}</span>
                </router-link>
           </template>	
          <p style="color: rgb(52, 89, 198);padding: 0px;padding-left: 0px;padding-left: 16px;font-weight: bold;margin-bottom: 2px;font-size: 14px;">Bookmarks:</p>
                <p class="nav" style="border-top: solid 2px #c6c6c6; padding: 0px !important;background-color: rgb(248, 248, 248); margin-top: 0px !important; margin-bottom: 0px !important;font-size: 14px !important;">
                <span>
		          <a style="padding: .2em;" class="action" href="http://192.168.0.2/files/mnt/cache/appdata/" ><i style="margin-right: -34px; color: rgb(52, 89, 198);" class="material-icons">bookmarks</i>appdata</a>
	          </span>
              </p>
                <p class="nav" style="padding: 0px !important;background-color: rgb(248, 248, 248); margin-top: 0px !important; margin-bottom: 0px !important;font-size: 14px !important;" >
                <span>
		          <a style="padding: .2em;" class="action" href="http://192.168.0.2/files/mnt/cache/appdata/swag/www/" ><i style="margin-right: -34px;; color: rgb(52, 89, 198);" class="material-icons">bookmarks</i>www</a>
	          </span>
              </p>`

Obviously my hard-coded bookmarks are only good for me. It got me thinking how we can use the **branded folder** with the custom.css file to personalize. What if we are able to drop and edit a plain html file into this folder, so users can customize their bookmarks on this file.

I have tried looking how this could be done. Supposedly its easy, but what I found is beyond my programming skills. Would someone know what to insert in the **_Sidebar.vue_** file that would load an external html (located in same folder as custom.css)?
